source/options.ts

```diff
@@ -71,4 +71,6 @@ export interface DefaultAddOptions {
 	@default 0
 	*/
 	readonly priority?: number;
+
+	readonly [key: string]: unknown;
```

Comment:
Not sure if this is the right way to go, buuut if the idea is to satisfy `QueueAddOptions`, it might be better to just extend it?

Reply:
Good call 👍